docs/functions.rst

```diff
@@ -0,0 +1,98 @@
+Functions in the Tax and Transfer System
+========================================
+
+Allgemein
+---------
+
+.. autosummary::
+   :toctree: _generated
+
+   gettsim.demographic_vars
+
+
+Sozialleistungen
```

Comment:
Should we provide the English translation in parentheses?

Reply:
Hi @hmgaudecker. To avoid ambiguity, what do you think about a glossary for key terms and translations for English and German. Sphinx has a builtin for that. Here is an example https://respy.readthedocs.io/en/latest/glossary.html.

Reply:
Sounds great, but no top priority. Maybe provide a shell with one-two examples s.t. someone can complete it once things have settled down a bit.